index.html

```diff
@@ -1007,7 +1011,11 @@ <h3>Algorithm</h3>
         <li>
           For each item <var>context</var> in <var>local context</var>:
           <ol>
-            <li>If <var>context</var> is <code>null</code>, set <var>result</var> to a
+            <li>If <var>context</var> is <code>null</code>
+              <span class="changed">and <var>from sealed term</var> is <code>true</code>
+                  or the <var>active context</var> contains no <a>term definition</a>
```

Comment:
This should be "IF context is null  AND ( 'from sealed term' is true OR the active context contains... )"
with those parenhthesis.
The current phrasing does not make the parenthesis explicit, and it might be misleading because the usual precedence of AND and OR implies something else.

PROPOSED: "IF context is null  AND EITHER 'from sealed term' is true OR the active context contains..."

Reply:
Comment made irrelevant by subsequent changes.